asv_bench/benchmarks/algorithms.py

```diff
@@ -13,6 +15,19 @@
         pass
 
 
+class MaybeConvertObjects:
```

Comment:
Not sure we even need this benchmark since it doesn't indicate anything from end user experience but up to @jreback 

Break alone in PR lgtm

Reply:
I want to add more tests here later as well - for more generalized cases.